### DISCUSSION TITLE
Show differences within a word/text string

### DISCUSSION THREAD

**Discussion by jsparlaiolan** [WinMerge/winmerge#1688] [Q&A] [answered]
When comparing files, WinMerge marks whole words as different, even if only some characters differ.

I.e. when comparing 'test123' and 'test456', WinMerge marks both 'test123' and 'test456' as different, even in the line differences window, while in fact only the '123' and '456' parts differ.
![afbeelding](https://user-images.githubusercontent.com/123633132/214813218-062a1e5a-9b64-4d00-86fd-290f15c7da55.png)

Is there any way I can have WinMerge mark the line as different, and when checking the line differences window, only have it mark '123' and '456' when the 'test' part of the word is the same?

**Comment by sdottaka** [comment, marked as answer]
You can get the expected result by setting the following in the Options dialog box.

![image](https://user-images.githubusercontent.com/98126/214834537-a53640fd-7383-47ab-aa69-7b1aef804e5d.png)
